dags/examples/xpk_example_dag.py

```diff
@@ -31,14 +32,19 @@
     start_date=datetime.datetime(2023, 11, 29),
     catchup=False,
 ) as dag:
+  docker_image_build = task.DockerBuildTask(
+      build_dir="dags/examples/configs/build",
+      image_name=DockerImage.XPK_JAX_TEST.value,
+      custom_build=True,
```

Comment:
Sure, this XPK example is actually better-suited to the direct docker build, since it's just a Dockerfile.

@raymondzouu can I add a docker build step to your parameter sweep example to build MaxText?

Reply:
Actually, parameter sweep is probably the one type of XPK test where we don't want a docker build. I'll just add a new DAG.